catalog/core/catalog-core-api/src/main/java/ddf/catalog/source/SourceMonitor.java

```diff
@@ -14,28 +14,26 @@
 package ddf.catalog.source;
 
 /**
- * The Interface {@link SourceMonitor} is used as a callback object to set the availability for the
- * {@link ddf.catalog.util.SourcePollerRunner}
+ * The Interface {@link SourceMonitor} is used as a callback object for {@link
+ * Source#isAvailable(SourceMonitor)}.
  */
 public interface SourceMonitor {
```

Comment:
Updating the javadoc here because `SourceMonitor`s do not use `SourceStatus`es, and the `SourcePollerRunner` does not use `SourceMonitor`s